hackathon_site/static/event/js/application.js

```diff
@@ -0,0 +1,3 @@
+$(document).ready(function () {
```

Comment:
this folder hackathon_site/static should be gitignored, I don't know how this ended up here 🤔

but anyways, you should make these changes in hackathon_site/event/static/event/js/application.js

Reply:
Actually, did you manually add this file? I think that could be a way to break gitignore

Reply:
Yeah I did I can undo this file, but to clarify do you want me to commit the application.js file in hackathon_site/event/static/event/js/application.js?

Reply:
yes, correct